aci-preupgrade-validation-script.py

```diff
@@ -613,7 +618,10 @@ def icurl(apitype, query):
     logging.debug('response: ' + str(response))
     imdata = json.loads(response)['imdata']
     if imdata and "error" in imdata[0].keys():
-        raise Exception('API call failed! Check debug log')
+        if imdata and "not found in class" in imdata[0]['error']['attributes']['text']:
```

Comment:
```suggestion
        if "not found in class" in imdata[0]['error']['attributes']['text']:
```
Let's do exactly the same in the real icurl (this) and the mock icurl in pytest. There is no need to do `imdata and` again as it's already checked in the previous line. The mock icurl is not doing it either.